docs/quickstart.html

```diff
@@ -100,15 +100,15 @@
 <p>
 First we make a config file for each of the brokers (on Windows use the <code>copy</code> command instead):
 </p>
-<pre>
-&gt; <b>cp config/server.properties config/server-1.properties</b>
-&gt; <b>cp config/server.properties config/server-2.properties</b>
+<pre class="brush: bash;">
+&gt; cp config/server.properties config/server-1.properties
+&gt; cp config/server.properties config/server-2.properties
 </pre>
 
 <p>
 Now edit these new files and set the following properties:
 </p>
-<pre>
+<pre class="brush: text;">
```

Comment:
When we are showing the file content: `text`, when we are showing the cmdline: `bash` (`bash` and `shell` are the same alias).